CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+Signing HTTP Messages Contribution Guide
+========================================
+
+This specification is under active development, and the more participants
+we get the better for everyone. We want this specification to be a
+useful addition to the ecosystem of the Internet, so it needs to be
+readable, useful and effective.
+
+You can contribute to the project by looking at the
+[open issues](https://github.com/w3c-dvcg/http-signatures/issues) and join
+a discussion, or raise a new issue if you have a bug or new feature
+suggestion.
+
+First though, take a look at the guiding principles of the project:
+
+# Principles
+
+## Platform, Language & Transport neutrality
+
+This protocol is meant to provide a technical mechanism to produce and
+verify digital signatures of HTTP Messages in a platform-agnostic way
```

Comment:
iiuc this protocol wants to sign `HTTP header fields`. This is different from the HTTP Message which is subject to  Transfer-Coding  and mangling of intermediaries.

Reply:
See 7594164. Keen to not get overly specific in the guidelines, as this is covered in the spec.